update-dependency-pinning.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
-WITNESS=app/witness.gradle
+# this will just append new checksums and not remove unnecessary ones
 
-echo "" > $WITNESS
-./gradlew -q calculateChecksums | grep -Ev "^(Skipping|Verifying)" | grep -Ev "files-2.1:|caches:transforms-3:|:build-tools:core-lambda-stubs.jar:|:platforms:android.jar:|-linux.jar:" > $WITNESS
+# to clean up the file, remove ./gradle/verification-metadata.xml,
+# run the command below and manually (re-)add checksums for missing operating systems windows, osx or linux for aapt2
+# checksums can be computed after downloading the respective jars of https://maven.google.com/web/index.html?q=aapt2#com.android.tools.build:aapt2
 
+./gradlew --write-verification-metadata sha256 build
```

Comment:
Does this overwrite existing entries? What happens if a checksum has changed, e.g. now it gets a different one than was in that file before?

Reply:
I am not fully sure. I suspect it would add the new checksum as a second option. Looking at the diff will again reveal the behavior and one could manually react to it.
Btw, the gradle witness plugin would silently change the checksum and one would only notice it looking at the diff, too.